tests/integration/deploy/test_deploy_command.py

```diff
@@ -21,6 +21,7 @@
 SKIP_DEPLOY_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
 CFN_SLEEP = 3
 TIMEOUT = 300
+CFN_PYTHON_VERSION_SUFFIX = os.environ.get("PYTHON_VERSION", "3.6.9").replace(".", "-")
```

Comment:
Should we default to something that is not a valid Python version? This could be confusing to tell if there was no Python Version env var or the value of the env var was 3.6.9.

Reply:
Yeah, thats fair, this would go out of sync if python minor version is updated. I'll make that change.
Thanks